dot/network/sync.go

```diff
@@ -214,9 +216,11 @@ func (q *syncQueue) syncAtHead() {
 }
 
 func (q *syncQueue) handleResponseQueue() {
+	t := time.NewTicker(time.Second)
```

Comment:
can we make these durations configurable to the type, so `syncQueue` in this case?  It will help with testing and we can adjust the durations in the unit tests.

Reply:
good idea, I've added configuration variables.